emily/handler/src/api/models/withdrawal/requests.rs

```diff
@@ -122,28 +122,40 @@ impl UpdateWithdrawalsRequestBody {
     pub fn try_into_validated_update_request(
         self,
         chainstate: Chainstate,
-    ) -> Result<ValidatedUpdateWithdrawalRequest, error::Error> {
+    ) -> Result<ValidatedUpdateWithdrawalRequest, error::ValidationError> {
         // Validate all the withdrawal updates.
-        let mut withdrawals: Vec<(usize, ValidatedWithdrawalUpdate)> = vec![];
-        let mut failed_ids: Vec<u64> = vec![];
+        let mut withdrawals: Vec<(usize, Result<ValidatedWithdrawalUpdate, ValidationError>)> =
+            vec![];
 
         for (index, update) in self.withdrawals.into_iter().enumerate() {
             match update
                 .clone()
                 .try_into_validated_withdrawal_update(chainstate.clone())
             {
-                Ok(validated_update) => withdrawals.push((index, validated_update)),
-                Err(_) => failed_ids.push(update.request_id),
+                Ok(validated_update) => withdrawals.push((index, Ok(validated_update))),
+                Err(ref err @ ValidationError::WithdrawalMissingFulfillment(request_id)) => {
+                    tracing::warn!(
+                        request_id,
+                        "failed to update withdrawal: request missing fulfillment for completed request."
+                    );
+                    withdrawals.push((index, Err(err.clone())));
+                }
+                Err(err) => {
+                    tracing::error!(
+                        request_id = update.request_id,
+                        ?err,
```

Comment:
Same thing here, `?err` to `%error`

Reply:
[f36db4f](https://github.com/stacks-sbtc/sbtc/pull/1653/commits/f36db4fdbee7619b946abfa6a11be7ec4ee05c51)